src/Command/SystemImportCommand.php

```diff
@@ -29,6 +34,24 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->systemImporter->import($input->getArgument('src'));
+        $success = true;
+        $errorMessage = null;
+
+        try {
+            $this->systemImporter->import($input->getArgument('src'));
+        } catch (\Exception $e) {
+            $success = false;
+            $errorMessage = $e->getMessage();
+            $output->writeln($errorMessage);
+        }
+
+        $importRun = new ImportRun();
+        $importRun->setDatetime(new \DateTime());
+        $importRun->setOutput($success ? 'OK' : $errorMessage);
+        $importRun->setResult($success);
+        $importRun->setType(System::class);
+
+        $this->entityManager->persist($importRun);
+        $this->entityManager->flush();
```

Comment:
We could add an `ImportInterface` and a `AbstractImportCommand` to prevent duplicating this code …

Reply:
Good point. It will be done.